sr_hardware_interface/include/sr_hardware_interface/tactile_sensors.hpp

```diff
@@ -276,6 +296,12 @@ class BiotacData
   int tac;  // int16u in word[2]
   int tdc;  // int16u in word[2]
   std::vector<int16_t> electrodes;  // int16u in word[2]
+  boost::circular_buffer<int16_t> pac_buffer_;  // 2.2kHz history of int16u/word[2] values. Capacity of 270 samples is
+                                                // 10 read cycles, or 122ms history
```

Comment:
Not sure that I understand the numbers.
We read from the hand at 1KHz.
So 270 samples, when we collect 2 samples per cycle, correspond to 135 cycles == 135 ms of history.
can we clarify the comment and decide if the length makes sense for us?

Reply:
Clarified in https://github.com/shadow-robot/sr_core/pull/136/commits/bb41fb129decf14a2035232727e904319bafdba8